build.gradle

```diff
@@ -44,7 +44,7 @@ allprojects {
         }
 
         maven {
-            url = 'https://maven.pkg.github.com/ballerina-platform/ballerina-lang'
+            url = 'https://maven.pkg.github.com/ballerina-platform/*'
```

Comment:
Since there is only one url lets leave it as it is

Reply:
Fixed https://github.com/ballerina-platform/module-ballerina-io/pull/226/commits/3b3e42f9ce44792f4bb3e0fecf295f36422c6157